docker/build/dev.x86_64.dockerfile

```diff
@@ -2,37 +2,45 @@ FROM nvidia/cuda:8.0-cudnn7-devel-ubuntu14.04
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && \
-   apt-get install -y \
-   bc \
-   cppcheck \
-   debconf-utils \
-   doxygen \
-   graphviz \
-   gdb \
-   git \
-   subversion \
-   google-perftools \
-   lcov \
-   libblas-dev \
-   libboost-all-dev \
-   libcurl4-openssl-dev \
-   libfreetype6-dev \
-   liblapack-dev \
-   libpcap-dev \
-   locate \
-   lsof \
-   realpath \
-   shellcheck \
-   vim \
-   v4l-utils \
-   nfs-common \
-   zip && \
-   apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apt-get update -y && \
+    apt-get install -y \
+    apt-transport-https \
```

Comment:
Please make this list in sorted order. So that it's easier for future lookup and modification.

Reply:
thanks for your quick review, I sorted the list and pushed another commit on the same branch. Please review again.